README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/tfutils/tfenv.svg?branch=master)](https://travis-ci.org/tfutils/tfenv)
+[![Build Status](https://api.travis-ci.com/tfutils/tfenv.svg?branch=master)](https://travis-ci.com/tfutils/tfenv)
```

Comment:
might want to remove the `api`, this is the official url that's currently blessed by travis-ci

```suggestion
[![Build Status](https://travis-ci.com/tfutils/tfenv.svg?branch=master)](https://travis-ci.com/tfutils/tfenv)
```

Reply:
great!